examples/webhook_examples/webhook_cherrypy_echo_bot.py

```diff
@@ -73,6 +73,11 @@ def echo_message(message):
 bot.set_webhook(url=WEBHOOK_URL_BASE+WEBHOOK_URL_PATH,
                 certificate=open(WEBHOOK_SSL_CERT, 'r'))
 
+# Disable CherryPy requests log
```

Comment:
Thanks! I tried to find this solution, however all methods except this one didn't work.